### ISSUE TITLE
No sound after pause and resume

### ISSUE THREAD

**Issue by npv1459** [patrickl92/AudioChecklist#5]
Did a couple of flight with the Zibo 737-800 and tried different SOP.
If you pause a checklist and you resume it, it will continue without any sound. 
You need to select again a new SOP to ear again the voices.
Atteched the xplane log file.

I will also open another issue with the same log because there is another problem

[](url
[Log.txt](https://github.com/patrickl92/AudioChecklist/files/7913035/Log.txt)
)

**Comment by patrickl92** [comment]
I've just publish a new release, which allows you to enable debug loggin using the menu entry `Plugins > FlyWithLua > FlyWithLua Macros > Audio Checklist: Enable Debug Logging`. Please download this release and activate this option before starting the SOP. Do your flight as normal and if you encounter the problem again, please upload the X-Plane log file here. It is located in the X-Plane directory, named `Log.txt`.

Note that the debug logging will always be disabled per default when starting X-Plane or selecting another aircraft.

**Comment by patrickl92** [comment]
Closed due to inactivity
